packages/terra-form-select/examples/test-examples/ControlledDefault.example.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Base from 'terra-base';
```

Comment:
All example are wrapped in base by terra-dev-site so we can drop wrapping these in Base

Reply:
Removed